src/app/components/date-range-picker/index.html

```diff
@@ -0,0 +1,262 @@
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties
+    sectionHeading="SkyDateRangeCalculatorId properties"
+  >
+    <sky-demo-page-property
+      propertyName="After"
+    >
+      Enables users to select a start date with no end date.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="AnyTime"
+    >
+      Selects no dates and considers all dates within the date range. This is the default selection.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="Before"
+    >
+      Enables users to select an end date with no starting date.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="LastCalendarYear"
+    >
+      Sets the start date to the first day of the year before the current year and the end date to the last day of that year.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="LastFiscalYear"
+    >
+      Sets the start date to the first day of the fiscal year before the current fiscal year and the end date to the last day of that fiscal year. The fiscal year is Oct. 1 to Sept. 30.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="LastMonth"
+    >
+      Sets the start date to the first day of the month before the current month and the end date to the final day of that month.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="LastQuarter"
+    >
+      Sets the start date to the first day of the quarter before the current quarter and the end date to the last day of that quarter. Quarters are January to March, April to June, July to September, and October to December.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="LastWeek"
+    >
+      Sets the start date to Sunday of the week before the current week and the end date to Saturday of that week.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="NextCalendarYear"
+    >
+      Sets the start date to the first day of the year after the current year and the end date to the last day of that year.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="NextFiscalYear"
+    >
+      Sets the start date to the first day of the fiscal year after the current fiscal year and the end date to the last day of that fiscal year. The fiscal year is Oct. 1 to Sept. 30.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="NextMonth"
+    >
+      Sets the start date to the first day of the month after the current month and the end date to the last day of that month.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="NextQuarter"
+    >
+      Sets the start date to the first day of the quarter after the current quarter and the end date to the last day of that quarter. Quarters are January to March, April to June, July to September, and October to December.
```

Comment:
Should we link to a footnote or something similar that contains your explanation text for "Quarters" so we don't have to repeat ourselves?

Reply:
Resolving this based on our conversation at standup. I may return to this if I can think of an effective way to avoid repeating this content, but the options I've thought of so far don't work very well for this scenario.